tests/Foundation/FoundationInteractsWithMailableTest.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+use Mockery as m;
```

Comment:
`Mockery` get repeated several times in just a single test so it's a convenient to alias it as `m` later it became convention. :)